kubernetes/config-maps/concent/timing_settings.py.j2

```diff
@@ -1,7 +1,11 @@
 {% if fast_timing %}
-ADDITIONAL_VERIFICATION_CALL_TIME = 15
+CUSTOM_PROTOCOL_TIMES = True
```

Comment:
We needed to add this, because this was hardcoded in base.py and we couldn't change it during tests.